### DISCUSSION TITLE
lookup-service-level-3 : rest service problem?

### DISCUSSION THREAD

**Discussion by jonashaggstrom** [DevSkillsHQ/discusssions#48] [Help] [answered]
I get a 404 for accessing personal-details path with the ssn 424-11-9327

**Comment by jonashaggstrom** [comment]
I tried using get request with postman to https://infra.devskills.app/personal-details/424-11-9327

**Reply by jonashaggstrom** [reply, marked as answer]
I had the wrong base url so https://infra.devskills.app/api/credit-data/personal-details/424-11-9327 works!

**Comment by fenske** [comment]
Hi Jonas. I'm Anton from Alva. Glad you figured it out! I'll close this discussion in the meantime. Let me know if you have any more questions. 

P.S. With the Lookup Service challenge, we are trying out the new structure of the README. If you have a moment to share if it made sense and, in particular, if you tried the Codespaces setup, it'd be cool to know how it went for you.

Best of luck with the rest of the process!

**Reply by jonashaggstrom** [reply]
Yes, The readme file makes perfect sense. 
Yes, it was in fact easier and faster coding and running code in Codespaces than on local machine. The best part compared to running locally in Eclipse is that all the versions of jdk, npm etc are correct when you start in codespaces.
On local machine i had issues with a previous Java 19  not compatible with gradle version etc. 
The downside is that you get disconnected after a while in codespaces and it takes a few seconds to reconnect.
I have not experienced any other problems with codespaces but I have not used it before.
